notebooks/en/benchmarking_tgi.ipynb

```diff
@@ -0,0 +1,443 @@
+{
```

Comment:
<p>I think we can keep it more minimal here :)</p><p> </p><p>"The purpose of this cookbook is to show you how to benchmark TGI. For more background details and explanation, please check out this <a href="https://huggingface.co/blog/tgi-benchmarking" target="_blank">blog post</a> first."</p><p> </p><p>"...; Docker is a great choice. The commands here can be easily copied/pasted into a terminal, which might be even easier. Don't feel compelled to use Jupyter. If you just want to test this out, you can duplicate and use [derek-thomas/tgi-benchmark-space](<a href="https://huggingface.co/spaces/derek-thomas/tgi-benchmark-space" target="_blank">https://huggingface.co/spaces/derek-thomas/tgi-benchmark-space</a>).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/180/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Done</p>